archaius2-archaius1-bridge/src/main/java/com/netflix/archaius/bridge/StaticDeploymentContext.java

```diff
@@ -89,7 +89,7 @@ public void setDeploymentRegion(String region) {
     @Override
     public String getValue(ContextKey key) {
         if (delegate == null) {
-            LOG.warn("Configuration not yet initialized.  Returning 'null' for " + key);
+            LOG.debug("Configuration not yet initialized.  Returning 'null' for " + key);
```

Comment:
Just curious, why does this class use logging but StaticAbstractConfiguration is being changed to System.err?